osrelease/setup.sh

```diff
@@ -0,0 +1,20 @@
+#!/bin/bash
+# Set up the job-runner service
+set -euo pipefail
+
+DIR=/srv/osrelease
+mkdir -p $DIR
+# ensure we have a checkout of output-publisher
+test -d $DIR/code || git clone https://github.com/opensafely-core/output-publisher $DIR/code
+mkdir -p $DIR/environ
+
+secrets_env="$DIR/environ/config.env"
+test -f $secrets_env || cp osrelease/secrets-template.env $secrets_env
+
+# reviewers group need to run osrelease, so they need to be able to read
+# osrelease code and config
+chown -R root:reviewers $DIR
```

Comment:
would mildly question whether this needs to be `root` vs anything else, but these days I think we're assuming that if someone has a shell account on the server, they can probably find a hole somewhere to escalate via?

Reply:
Right, I just did root as I didn't both creating a shared account for osrelease, and I didn't want researchers to be able to write to it. Wasn't specifically wanting to restrict it to root for any other reason.